### PR TITLE
feat(demo): FRMW-60-rxToggleSwitch failed async demo

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-60 -  Demo: rxToggleSwitch Needs Example of Failed Async Operation

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

---
Added failed asynchronous operation example to `rxToggleSwitch` demo:
* User attempts to activate a service
* `rxToggleSwitch` shows disabled activation state as service attempts asynchronous operation
* Asynchronous operation fails
* `rxToggleSwitch` returns to deactivated state

### CODE REVIEW

**Comment by 100stacks** [comment]
Firefox 45: Demo - Initial State

screen shot 2015-10-21 at 6 09 01 pm

**Comment by 100stacks** [comment]
Firefox 45: Demo - Attempting asynchronous operation
- Please notice the disabled activation state

screen shot 2015-10-21 at 6 07 29 pm

**Comment by 100stacks** [comment]
Firefox 45: Demo - Asynchronous operation failed
- Please notice the service reverted to OFF position, and failed notification banner

screen shot 2015-10-21 at 6 07 43 pm

File: src/rxToggleSwitch/docs/rxToggleSwitch.js [hunk 1]
```diff
@@ -18,4 +20,22 @@ angular.module('demoApp')
...
+        });
+
+        // Simulate a failed API request
+        $timeout(function () {
+            $scope.loading5 = false;
+            rxNotify.clear('page');
+            rxNotify.add('Asynchronous operation failed', {
+                type: 'error',
+            });
+            $scope.toggle5 = !value;
```
Comment (CITguy) [inline src/rxToggleSwitch/docs/rxToggleSwitch.js]: I'd put a comment here that this is to reset the toggle switch to its initial value.

**Comment by 100stacks** [comment]
@CITguy comment added :+1:

File: src/rxToggleSwitch/docs/rxToggleSwitch.js [hunk 2]
```diff
@@ -18,4 +20,22 @@ angular.module('demoApp')
...
+        });
+
+        // Simulate a failed API request
+        $timeout(function () {
+            $scope.loading5 = false;
+            rxNotify.clear('page');
+            rxNotify.add('Asynchronous operation failed', {
+                type: 'error',
+            });
+            $scope.toggle5 = !value;// Reset toggle switch to original value to simulate failed async operation
```
Comment (CITguy) [inline src/rxToggleSwitch/docs/rxToggleSwitch.js]: please place comment above code. sorry for the confusion

**Comment by bry4n** [comment]
LGTM

**Comment by CITguy** [comment]
Functionality verified.

failed-async-rxtoggleswitch

**Comment by 100stacks** [comment]
@CITguy my bad...moved the comment :+1:

**Comment by 100stacks** [comment]
@CITguy what did you use to make the .gif video?

**Comment by CITguy** [comment]
@100stacks: http://www.cockos.com/licecap/

**Comment by CITguy** [comment]
LGTM

**Comment by 100stacks** [comment]
@CITguy Thanks!  The other tools I looked into required a series of screenshots stitched together.

**Comment by ssalinas24** [comment]
LGTM
Keeping in mind that the rest of the toggle switch error styling and solution will be covered in FRMW-69. https://jira.rax.io/browse/FRMW-69

**Comment by 100stacks** [comment]
@ssalinas24 okay rebased :+1:

**Pull request merged**